flow-typed/npm/array-range_vx.x.x.js

```diff
@@ -8,8 +8,8 @@
  *
  * Fill this stub out by replacing all the `any` types.
  *
- * Once filled out, we encourage you to share your work with the 
- * community by sending a pull request to: 
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
  * https://github.com/flowtype/flow-typed
  */
```

Comment:
some changes are really useless, that's been automatically done by the flow-typed CLI, tell me if you'd like me to remove them.